src/posthog-core.ts

```diff
@@ -915,6 +924,13 @@ export class PostHog {
         let properties = { ...event_properties }
         properties['token'] = this.config.token
 
+        if (this.config.__preview_experimental_cookieless_mode) {
+            // Set a flag to tell the plugin server to use cookieless server hash mode
+            // on naming: $cklsh = cookieless server hash
+            // I didn't call it $cookieless, as it's possible to use cookies in this mode (after consent is given)
+            properties['$cklsh'] = true
```

Comment:
Given this is a magic prop could we move it to constants and give it a nice name?

Reply:
Done